gradle/publish.gradle

```diff
@@ -1,34 +1,8 @@
-/*
```

Comment:
We are extending opensourced client, thus there will be upstream changes from time to time.
As long as it does not affect result - I would leave as much of lines as possible intact.

And perhaps extend S3 way of publishing as a separate gradle publishing task.
Who knows, maybe version will be published to artifactory as well.

Reply:
Added Artifactory/Bintray along with S3 Maven repo